### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by jwmiller64** [john-stephens/zebra-mac-label-automator#7] [General]
I had given up on ever getting readable barcodes from an old  GX420d printer. Thanks to your work I now can. Your instructions were perfect. It is installed on an iMac (2021) M1 chip running Sonoma 14.6.1

**Comment by john-stephens** [comment]
Awesome! Glad it's working for you.

**Comment by jwmiller64** [comment]
After finding your solution, I came across
https://www.msweet.org/lprint/lprint.html. He is one of the authors of the
cups printing system that Apple uses. His Lprint uses cups to directly
print to the label printer eliminating the need for the image
manipulation.
Thank you for your solution as well.

On Tue, Nov 19, 2024 at 2:54 PM John Stephens ***@***.***>
wrote:

> Awesome! Glad it's working for you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/john-stephens/zebra-mac-label-automator/discussions/7#discussioncomment-11311827>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQTWU5HBOLY4ZRU3U36L7ST2BOQPJAVCNFSM6AAAAABSC2UI2WVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMZRGE4DENY>
> .
> You are receiving this because you authored the thread.Message ID:
> <john-stephens/zebra-mac-label-automator/repo-discussions/7/comments/11311827
> @github.com>
>


-- 
John Miller
Flexible Plastics, Inc.             T 507.625.1681
301 Maxfield St                      T 800.633.4109
Mankato MN 56001                F 507.625.1168
jmiller(at)flexibleplastics.com
